egcg_core/app_logging.py

```diff
@@ -2,27 +2,39 @@
 import logging
 import logging.config
 import logging.handlers
-from egcg_core.config import default as cfg
+from egcg_core.config import cfg
 
 
 class LoggingConfiguration:
     """Stores Loggers, Formatters and Handlers"""
 
     default_fmt = '[%(asctime)s][%(name)s][%(levelname)s] %(message)s'
     default_datefmt = '%Y-%b-%d %H:%M:%S'
+    _formatter = None
```

Comment:
I'm not sure what the use of private variable and property decorator is achieving here